src/sql/workbench/services/objectExplorer/browser/asyncServerTreeRenderer.ts

```diff
@@ -70,12 +70,15 @@ class ConnectionProfileGroupTemplate extends Disposable {
 			matches: createMatches(filterData)
 		});
 
-		const actionProvider = this._objectExplorerService.getServerTreeView().treeActionProvider;
-		const tree = this._objectExplorerService.getServerTreeView().tree;
-		const actions = actionProvider.getActions(tree, element, true);
-		this._actionBar.context = this._objectExplorerService.getServerTreeView().getActionContext(element);
-		this._actionBar.clear();
-		this._actionBar.pushAction(actions, { icon: true, label: false });
+		let serverTreeView = this._objectExplorerService.getServerTreeView();
+		if (serverTreeView) {
```

Comment:
Maybe we should throw an error message for the else case for each of these checks?